src/main/kotlin/apply/utils/MarkdownProcessor.kt

```diff
@@ -0,0 +1,14 @@
+package apply.utils
+
+import org.intellij.markdown.flavours.commonmark.CommonMarkFlavourDescriptor
+import org.intellij.markdown.html.HtmlGenerator
+import org.intellij.markdown.parser.MarkdownParser
+
+object MarkdownProcessor {
+    private val flavour = CommonMarkFlavourDescriptor()
```

Comment:
r: 타입 선언이 빠졌습니다. 지역 변수를 제외한 모든 곳에 타입을 선언해 주세요.

```suggestion
    private val flavour: MarkdownFlavourDescriptor = CommonMarkFlavourDescriptor()
```